java/com/epam/ta/salad/Repository/util/VegetableFieldsValidator.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.ta.salad.Repository.util;
+
+import com.epam.ta.parking.util.NullCheckUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class VegetableFieldsValidator {
```

Comment:
First of all, 'static' isn't a good practice, it uses in rare cases.
Secondary, you can make every method private, and write one public, which call all private.

Reply:
Also you can change the logic of validator. Now you return the result of validation, an entry is correct or not, but change it to return a list with incorrect fields.

Reply:
seems, fixed, but I don't understand for which purpose

Reply:
![image](https://user-images.githubusercontent.com/46902048/69721791-a9989500-1126-11ea-8df3-6509dcab9991.png)
This change in validator makes me feel confused about this one: how I can change this this to overcome this error?